aesara/tensor/random/basic.py

```diff
@@ -492,9 +492,10 @@ def rng_fn(cls, rng, n, a, b, size=None):
 class MultinomialRV(RandomVariable):
     """A Multinomial random variable type.
 
-    FYI: Support shape is determined by the first dimension in the *second*
-    parameter (i.e.  the probabilities vector).
-
+    Notes
+    -----
+    The length of the support dimension is determined by the last
+    dimension in the *second* parameter (i.e.  the probabilities vector).
```

Comment:
Which second parameter?

Reply:
The probabilities vector is the second parameter of the `MultinomialRV`.